frontend/vite.config.ts

```diff
@@ -2,4 +2,15 @@ import { viteConfigs } from '@gear-js/frontend-configs';
 import tailwindcss from '@tailwindcss/vite';
 import { defineConfig, mergeConfig } from 'vite';
 
-export default mergeConfig(viteConfigs.app, defineConfig({ plugins: [tailwindcss()] }));
```

Comment:
We can keep the original config here, no need for changes if `vite --host localhost` is working.